packages/ui/src/app/pages/Proposals/CurrentProposals.stories.tsx

```diff
@@ -781,6 +781,46 @@ export const SpecificParametersFundingRequest: Story = {
   }),
 }
 
+export const SpecificParametersMultipleFundingRequest: Story = {
+  play: specificParametersTest('Funding Request', async ({ args, createProposal, modal, step }) => {
+    const bob = member('bob')
+    await createProposal(async () => {
+      const nextButton = getButtonByText(modal, 'Create proposal')
+      expect(nextButton).toBeDisabled()
+
+      await userEvent.click(modal.getByTestId('pay-multiple'))
+
+      const csvField = modal.getByTestId('accounts-amounts')
+
+      // Invalid
+      await userEvent.clear(csvField)
+      await userEvent.type(csvField, `${alice.controllerAccount},500${bob.controllerAccount},500`)
+      expect(modal.findByText(/Not valid CSV format/))
+      expect(nextButton).toBeDisabled()
```

Comment:
Please also make sure the `PreviewAndValidateModal` can't be opened at this point.